<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>netmanager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/pixmaps/netmanager/tong.jpg</normaloff>../../../../usr/share/pixmaps/netmanager/tong.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>230</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Default Gateway</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Host Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtGateWay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRestartNetwork">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>230</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart Net</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DNS1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DNS2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDNS1">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>170</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDNS2">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>200</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>230</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>If dhcp,keep default gateway clean</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>601</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabwire">
     <attribute name="title">
      <string>cable network</string>
     </attribute>
     <widget class="QTableWidget" name="tableEthe">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>131</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>661</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Card Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DHCP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mask</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gateway</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Boot</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabwireless">
     <attribute name="title">
      <string>wireless network</string>
     </attribute>
     <widget class="QTableWidget" name="tableWireless">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>131</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>661</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>ESSID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Signal Strength</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnScanWireless">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan Wireless...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLink">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>100</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Link to Selected Net</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelWirelessCard">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>341</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Wireless Card:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPassword">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>74</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>net password(clean is null)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAsc">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>70</y>
        <width>87</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ASCII</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbWirelessType">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>WEP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WPA-PSK-TKIP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WPA-PSK-AES</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>105</y>
        <width>59</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="SaveAction"/>
    <addaction name="RestartNetworkCardAction"/>
    <addaction name="RefreshAction"/>
    <addaction name="HideAction"/>
    <addaction name="separator"/>
    <addaction name="QuitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="VersionAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="SaveAction">
   <property name="text">
    <string>Save the setting</string>
   </property>
   <property name="toolTip">
    <string>Save the setting</string>
   </property>
  </action>
  <action name="RestartNetworkCardAction">
   <property name="text">
    <string>Restart network card</string>
   </property>
   <property name="toolTip">
    <string>Restart network card</string>
   </property>
  </action>
  <action name="RefreshAction">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="VersionAction">
   <property name="text">
    <string>Get version</string>
   </property>
   <property name="toolTip">
    <string>Get version</string>
   </property>
  </action>
  <action name="HideAction">
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="toolTip">
    <string>Hide</string>
   </property>
  </action>
  <action name="QuitAction">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="ScanWirelessAction">
   <property name="text">
    <string>Scan wireless</string>
   </property>
   <property name="toolTip">
    <string>Scan wireless</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>SaveAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRestartNetwork</sender>
   <signal>clicked()</signal>
   <receiver>RestartNetworkCardAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRefresh</sender>
   <signal>clicked()</signal>
   <receiver>RefreshAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnScanWireless</sender>
   <signal>clicked()</signal>
   <receiver>ScanWirelessAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
